include/zephyr/bluetooth/gatt.h

```diff
@@ -72,6 +72,32 @@ enum {
 	 *  passed to write callback.
 	 */
 	BT_GATT_PERM_PREPARE_WRITE = BIT(6),
+
+	/** @brief Attribute write permission with 128-bit entropy key connection.
```

Comment:
read

Reply:
This change is from https://github.com/zephyrproject-rtos/zephyr/pull/46923

@MariuszSkamra I guess this should probably have been marked as a draft. I'll add the DNM until https://github.com/zephyrproject-rtos/zephyr/pull/46923 is merged.